wdl2cwl/old_main_oop.py

```diff
@@ -0,0 +1,470 @@
+"""Main entrypoint for WDL2CWL."""
```

Comment:
Good for your reference on your computer, but maybe shouldn't be part of this PR?